freqtrade/freqai/freqai_interface.py

```diff
@@ -637,6 +659,56 @@ def train_timer(self, do='start'):
                 self.train_time = 0
         return
 
+    def backtest_prediction_exists(
+        self,
+        pair: str,
+        dk: FreqaiDataKitchen,
+        trained_timestamp: int,
+        scanning: bool = False,
+    ) -> bool:
+        """
+        Given a pair and path, check if a backtesting prediction already exists
+        :param pair: pair e.g. BTC/USD
+        :param path: path to prediction
+        :return:
+        :boolean: whether the prediction file exists or not.
+        """
+        if not self.live:
+            prediction_file_name, root_prediction = self.get_backtesting_prediction_file_name(
+                pair, dk, trained_timestamp
+            )
+            path_to_predictionfile = Path(dk.full_path / root_prediction / prediction_file_name)
```

Comment:
e.g. `path_to_predictionfile = Path(dk.full_path / dk.backtesting_results_path / dk.model_filename)`

Reply:
Thanks for the feedback. Fixed!